generator/config/search/facet.yaml

```
@@ -0,0 +1,56 @@
+# $schema: ../schema.json
+name: facet
+link: 'https://www.mongodb.com/docs/atlas/atlas-search/facet/'
+type:
+    - searchOperator # should be searchCollector
+encode: object
+description: |
+    The facet collector groups results by values or ranges in the specified
+    faceted fields and returns the count for each of those groups.
+arguments:
+    -
+        name: facets
+        type:
+            - object # of facetDefinition
```

You previously used "of <type>" for an `array`. Was this intentional?

I assume it may be because it's a hash of facet names to definitions. Maybe change this to "# map of facetDefinition"?

---

Updated, thanks.
This comments are a kind of todo list to improve typing.
```suggestion
            - object # map of facetDefinition
```